assets/metrics-server/deployment.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: metrics-server
+    app.kubernetes.io/managed-by: cluster-monitoring-operator
+    app.kubernetes.io/name: metrics-server
+    app.kubernetes.io/part-of: openshift-monitoring
+  name: metrics-server
+  namespace: openshift-monitoring
+spec:
+  minReadySeconds: 60
```

Comment:
q: In enhancement proposal we have mentioned "Wait for `5 x metrics-server's scrape interval` to have enough data populated in `metrics-server` in-memory`" to switch APIService

` --metric-resolution` set is `15s`,  do we want to make this 75 seconds instead of 60 seconds?

Reply:
From https://github.com/kubernetes-sigs/metrics-server/pull/768, the readiness probe should return OK only when there are enough samples in memory so it's fine to remove `minReadySeconds` IMHO.